<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>attribute_table_widget</class>
 <widget class="QWidget" name="attribute_table_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>配置(C):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="configuration_combo"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>平台(P):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="platform_combo"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>视图(V):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="view_combo"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5">
     <item>
      <widget class="QScrollArea" name="attr_area">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>130</width>
          <height>218</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="attrs"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="attr_content_area">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>312</width>
          <height>226</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QStackedWidget" name="attr_views">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="muti_lines_page">
            <layout class="QGridLayout" name="gridLayout">
             <item row="3" column="0">
              <widget class="QTextEdit" name="alter_muti_lines_text"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>多行内容</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QTextEdit" name="original_muti_lines_text"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>多行内容编辑</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="raw_page">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>单行内容</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="original_text"/>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>单行内容编辑</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="alter_text"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="save_btn">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_as_btn">
       <property name="text">
        <string>另存为</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_nosave_btn">
       <property name="text">
        <string>不保存关闭</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_path_btn">
       <property name="text">
        <string>添加附加目录/附加依赖项</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
